README.md

```diff
@@ -346,6 +368,17 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [Good first issues](https://github.com/search?q=meilisearch+label%3Ahacktoberfest+state%3Aopen+is%3Aissue&type=Issues)
 - **Notes**: Check out [MeiliSearch blogpost](https://blog.meilisearch.com/contribute-hacktoberfest-2021/) for more details
 
+#### **Milvus**
+
+- **Swag**: a sticker pack, a digital Milvus contributor badge, and a Milvus T-shirt
+- **Requirements**: Sign up for the event on the [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/)  and just start contributing to any open source project - it’s as simple as that! If you’re new to the open source world, no worries, we’ve got you covered with [Guide](https://discuss.milvus.io/t/join-hacktoberfest-2021-with-us/72).
+  - By contributing to the Milvus project between October 1 and October 31 you’ll also earn some additional swag and prizes sponsored by Zilliz
+- **How to sign up**:
+  - Sign up [here](https://milvus.io/hacktoberfest-2021/)
+  - Join Discussions by creating [Milvus Forum Account](https://discuss.milvus.io/c/hacktoberfest/9) and interact with community members.
+- **Issues**: Issues are stated at Issues Section of [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/).
+- **Notes**: Check out [Milvus Community Forum](https://discuss.milvus.io/c/hacktoberfest/9)
+
```

Comment:
Following the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing), I'd love to see you format this entry again. Use the examples from line 280-287 above.